src/App.tsx

```diff
@@ -28,15 +38,84 @@ const MapListScreen = () => {
   return <MapList></MapList>;
 };
 
+//Created dummy About Us screen to test all three tabs
+const AboutUsScreen = () => {
+  return (
+    <View
+      style={{
+        flex: 1,
+        alignItems: "center",
+        justifyContent: "center",
+        backgroundColor: "#ffffff",
+      }}
+    >
+      <Text style={{ fontSize: 30 }}>About Us</Text>
+    </View>
+  );
+};
+
+// If the focused route is not found, we need to assume it's the initial screen
+// This can happen during if there hasn't been any navigation inside the screen
+// In our case, it's "Home" as that's the first screen inside the navigator
+const getHeaderTitle = (route) => {
+  const routeName = getFocusedRouteNameFromRoute(route) ?? "Home";
+
+  switch (routeName) {
+    case "Home":
+      return "Home";
+    case "Map":
+      return "MapList";
+    case "About Us":
+      return "About Us";
+  }
+};
+
+const Tabs = ({ navigation, route }) => {
+  React.useLayoutEffect(() => {
+    navigation.setOptions({ headerTitle: getHeaderTitle(route) });
+  }, [navigation, route]);
+
+  return (
+    <Tab.Navigator
+      screenOptions={({ route }) => ({
+        headerShown: false,
+        tabBarStyle: { backgroundColor: "black" },
+        tabBarIcon: ({ color, size }) => {
+          let iconName;
+
+          if (route.name === "Home") {
+            iconName = "home";
+          } else if (route.name === "Map") {
+            iconName = "map";
+          } else if (route.name === "About Us") {
+            iconName = "information-circle-outline";
+          }
+
+          // You can return any component that you like here!
+          return <Ionicons name={iconName} size={size} color={color} />;
+        },
+        tabBarActiveTintColor: "#ffc529",
+        tabBarInactiveTintColor: "#ffffff",
+      })}
+    >
+      <Tab.Screen name="Home" component={HomeScreen} />
+      <Tab.Screen name="Map" component={MapListScreen} />
+      <Tab.Screen name="About Us" component={AboutUsScreen} />
+    </Tab.Navigator>
+  );
+};
+
 const App = () => {
   return (
     <NavigationContainer>
       <Stack.Navigator>
-        <Stack.Screen name="Home" component={HomeScreen} />
+        <Stack.Screen name="Home" component={Tabs} />
```

Comment:
Try this approach instead: 
`<NavigationContainer>
      <Stack.Navigator initialRouteName="Tabs">
        <Stack.Screen name="Home" component={HomeScreen} />
        <Stack.Screen name="MapList" component={MapListScreen} />
        <Stack.Screen name="About Us" component={AboutUsScreen} />
        <Stack.Screen name="Tabs" component={TabNavigator} />
      </Stack.Navigator>
    </NavigationContainer>`
    
    This should also solve the nested screen name issue. Nice work!

Reply:
Replaced this code with the existing code and working fine without any warning. Thanks!